scalatest/src/main/scala/org/scalatest/Shell.scala

```diff
@@ -401,7 +401,7 @@ private[scalatest] final case class ShellImpl(
   lazy val nostacks: Shell = copy(shortstacksPassed = false, fullstacksPassed = false)
   lazy val nostats: Shell = copy(statsPassed = false)
 
-  def run(suite: Suite, testName: String = null, configMap: ConfigMap = ConfigMap.empty): Unit = {
+  def run(suite: Suite, testName: String | Null = null, configMap: ConfigMap = ConfigMap.empty): Unit = {
```

Comment:
**Nullable parameter**
Parameter that can be null and should therefore have a `| Null` type.